bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCommandWorkItem.java

```diff
@@ -228,4 +228,21 @@ public String toString() {
     public String workItemName() {
         return "pr-command";
     }
+
+    /**
+     * This method returns all the comments in the pr including comments in reviews(review body) and file specific comments
+     */
+    private List<Comment> getAllComments() {
+        // Regular comments
+        List<Comment> comments = new ArrayList<>(prComments());
+        // Comments in reviews
+        comments.addAll(pr.reviews().stream()
+                .map(review -> new Comment("Review" + review.id(), review.body().orElse(""), review.reviewer(), review.createdAt(), null))
+                .toList());
+        // File specific comments
+        comments.addAll(pr.reviewComments().stream()
+                .map(reviewComment -> new Comment("ReviewComment" + reviewComment.id(), reviewComment.body(), reviewComment.author(), reviewComment.createdAt(), reviewComment.updatedAt()))
+                .toList());
```

Comment:
I am not sure if the "review", "review comment", and "comment" in Github refer to the same numbering system, but based on my experiments, it is likely that they have their own independent numbering system. Although the probability of ID conflicts is small, I think it would be better to add "review" before the ID of a review, and "ReviewComment" before the ID of a review comment.

Reply:
Have you found a conflict? I did a quick check and didn't find any, but that's definitely no guarantee.

Reply:
I thought some more on your proposed solution here and I think it's sound.